### ISSUE TITLE
[Rotarycraft] Creative search does not work for several blocks

### ISSUE THREAD

**Issue by koravel** [ReikaKalseki/Reika_Mods_Issues#3160]
Blocks from RotaryCraft are not searchable via creative menu:
![image](https://user-images.githubusercontent.com/26851016/232250612-e7581f38-7c4e-41f4-9178-c18a2924a9d5.png)
![image](https://user-images.githubusercontent.com/26851016/232250620-7124645b-fa9b-47b6-9481-0a630af0f098.png)
![image](https://user-images.githubusercontent.com/26851016/232250669-17e54a35-8277-4ea8-a207-9291ad01d2c8.png)

**Comment by rogerwim** [comment]
I have this issue too, it's very annoying

**Comment by ReikaKalseki** [comment]
This is bizarre, and while I can reproduce it, it is pretty insignificant given that NEI search *does* work.

**Comment by koravel** [comment]
@ReikaKalseki at most cases it is not a big deal, unless some 'cool' mod uses creative search in their menus, like VendingBlock does

**Comment by rogerwim** [comment]
> This is bizarre, and while I can reproduce it, it is pretty insignificant given that NEI search *does* work.

Yes, but it made me think I installed mods wrong or screwed something up so many times, and then I have to add yet another mod to every single pack I use for testing
